.github/workflows/admission_webhook_test.yaml

```diff
@@ -24,6 +24,11 @@ jobs:
     - name: Create KinD Cluster
       run: kind create cluster --config tests/gh-actions/kind-cluster.yaml
 
+    - name: Install Calico
+      run: |
+        chmod +x ./tests/gh-actions/install_calico.sh
+        ./tests/gh-actions/install_calico.sh
```

Comment:
No reason to change the file modes and leaving the git work tree in a "dirty" state. You could just run `chmod +x ./tests/gh-actions/install_calico.sh`, commit that to the repository (like what is done with the other scripts in `tests/gh-actions/`), and use 

```yaml
run: ./tests/gh-actions/install_calico.sh`
```

As an alternative, you can skip the `chmod +x` entirely, and just run the Bash script using `bash`:

```yaml
run: bash ./tests/gh-actions/install_calico.sh
```

I personally prefer this, since I am not a fan of committing executables in git repositories, but I guess that the former is the preferred way in Kubeflow :man_shrugging: